src/reducers/user.ts

```diff
@@ -1,16 +1,15 @@
-import {fromJS} from 'immutable';
-import {EDIT_USER, LOGOUT, SET_USER} from '../constants/actions';
+import { fromJS } from "immutable";
+import { EDIT_USER, LOGOUT, SET_USER } from "../constants/actions";
 
 export const initialState = fromJS({
   id: null,
-  email: null
+  email: null,
```

Comment:
Wollen wir trailing commas rausnehmen? Wäre ich ein großer Fan von

Reply:
Ja, gerne. Normalerweise sollte das der Fall sein. Habe eine `.prettierrc` hinzugefügt, da er scheinbar deswegen "komisch" formatiert.

Werde einen Testcommit machen.

Reply:
Seit einiger Zeit ist das wohl von 'none' auf 'es5' gestellt als default, also überall ein komma wo es valide ist

Reply:
Ja, gesehen. Sie haben sich mit dem v2-Release wohl umentschieden. Habe es daher in der config auf `none` gestellt.